Gems/RecastNavigation/Code/Source/Misc/RecastNavigationPhysXProviderConfig.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/ComponentBus.h>
+#include <AzCore/RTTI/RTTI.h>
+#include <AzCore/Serialization/EditContext.h>
+
+namespace RecastNavigation
+{
+    class RecastNavigationPhysXProviderConfig final
```

Comment:
nit: An API comment for this would likely be useful

Reply:
Added.